inc/Integrations/SalesforceB2C/SalesforceB2CDataSource.php

```diff
@@ -21,6 +21,8 @@ protected static function get_service_config_schema(): array {
 			'enable_blocks' => Types::nullable( Types::boolean() ),
 			'organization_id' => Types::string(),
 			'shortcode' => Types::string(),
+			// ToDo: Remove the nullable existing SFCC data sources have been migrated.
+			'site_id' => Types::nullable( Types::string() ),
```

Comment:
This is set to nullable only until the existing test sites are migrated. After that, the nullable part can be dropped.

Same thing as below, it's an interim fix to add the `site_id` in place with the intention that the internal plan to add migration would negate this in the future.